### ISSUE TITLE
Horizontal lists work poorly due to a jQuery UI bug.

### ISSUE THREAD

**Issue by nateroling** [angular-ui/ui-sortable#19]
Horizontal sortable lists work poorly. This is because jQueryUI doesn't detect them as being horizontal.

Due to a bug in jQueryUI (http://bugs.jqueryui.com/ticket/7498), empty lists will never be considered horizontal.

When the sortable is created by Angular, the list is always empty. Thus, never detected as horizontal.

As an ugly workaround, you can set up a start listener to set the floating property manually, and sorting will work smoothly again:

``` javascript
 $scope.sortableOptions = {
    start: function(e, ui) {
      $(e.target).data("ui-sortable").floating = true;
    },
  };
});
```

``` html
<div ng-model='items' ui-sortable='sortableOptions'>
```

**Comment by thgreasi** [comment]
Added a notice to README with a2584246123c866bf16642ccb747cfa5b0a48dc5
since [~~jquery-ui issue  7498~~](http://bugs.jqueryui.com/ticket/7498) was resolved in jquery-ui v1.11.4.
Thanks for all your contributions in this issue.

**Comment by alexandre-le-borgne** [comment]
Thanks !! Not resolved in jquery ui 1.11.13
